shared/rakefiles/xk_config.rake

```diff
@@ -47,3 +65,16 @@ end
 task :configure_current_project => [@gcp_creds_file] do
   sh "gcloud config set project $TF_VAR_project_id"
 end
+
+task :set_auth_user_vars => [@gcp_creds_file] do
+  # Setting authenticated user's email into env variable, so it can be
+  # accessible in modules: https://issues.gpii.net/browse/GPII-3516
+  ENV['TF_VAR_auth_user_email'] = %x{
+    gcloud auth list --filter='account!~gserviceaccount.com' --format json |  jq -r '.[].account'
+  }.chomp!
+
+  # We build auth user SA from email by replacing non-allowed characters
+  # SA name length limit is 30 symbols
+  @auth_user_sa_name = ENV['TF_VAR_auth_user_email'].sub("@", "-at-").sub(".", "-")[0..30]
```

Comment:
Replacing `-` with `.` unconditionally seems dangerous.

Reply:
Dots are not allowed in SA name.